packages/keystone/runtimeStats.js

```diff
@@ -0,0 +1,208 @@
+const crypto = require('crypto')
+
+const express = require('express')
+
+const conf = require('@open-condo/config')
+const { getLogger } = require('@open-condo/keystone/logging')
+const metrics = require('@open-condo/keystone/metrics')
+
+const { DEFAULT_HEALTHCHECK_URL } = require('./healthCheck')
+
+const SEND_METRICS_INTERVAL_IN_MS = 1000
+
+const X_TARGET_OPTIONS_VAR_NAME = 'X_TARGET_OPTIONS'
+const ACCESS_TOKEN_VAR_NAME = 'RUNTIME_STATS_ACCESS_TOKEN'
+const ENABLED_VAR_NAME = 'RUNTIME_STATS_ENABLE'
```

Comment:
Please use the same const name as value it’s help to read code easily

Reply:
Renamed